index.html

```diff
@@ -423,7 +423,17 @@ <h3>Firing events using the <code>PointerEvent</code> interface</h3>
                 <p>If the event is {{GlobalEventHandlers/pointerdown}}, the associated device is a direct manipulation device, and the target is an {{Element}},
                     then <a>set pointer capture</a> for this <code>pointerId</code> to the target element as described in <a>implicit pointer capture</a>.
 
-                <p>Fire the event to the determined target. The user agent SHOULD treat the target as if the pointing device has moved over it for the purpose of <a data-cite="uievents/#events-mouseevent-event-order">ensuring event ordering</a> [[UIEVENTS]].</p>
+                <p>Fire the event to the determined target.
+                    The user agent SHOULD treat the target as if the pointing device has moved over it from the |previous target| (or if the |over child| flag is set, over an anonymous child of |previous target|)
+                    for the purpose of <a data-cite="uievents/#events-mouseevent-event-order">ensuring event ordering</a> [[UIEVENTS]].</p>
+
+                <p>Save the determined target as the |previous target| for the given pointer,
+                    and reset the |over child| flag to <code>false</code>.
+                    If the |previous target| at any point will no longer be [=connected=] [[DOM]],
+                    update the |previous target| to the nearest still [=connected=] [[DOM]] parent
+                    following the event path corresponding to dispatching events to the previous target,
```

Comment:
Note that this following the event path part is a bit hand wavy. Maybe we need to expose a method which encapsulates step 5.9 of [dispatching events](https://dom.spec.whatwg.org/#dispatching-events). One of the complications to doing that is the [get the parent](https://dom.spec.whatwg.org/#get-the-parent) algorithm can depend on the particular event being dispatched. I think it is the case that for all pointer events (and mouse events) the path is expected to be the same, but technically nothing stops a particular event from having a different path.